File: PigFarm/Models/Nutrition.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace PigFarm.Models
{
    public partial class Nutrition
    {
        public decimal Id { get; set; }
        public string NutritionType { get; set; }
        public string NutritionNo { get; set; }
        public string NutritionName { get; set; }
        public string NutritionElement { get; set; }
        public string NutritionEffect { get; set; }
        public string NutritionSideEffect { get; set; }
        public string NutritionBreed { get; set; }
        public string NutritionRange { get; set; }
        public string NutritionCare { get; set; }
        public string Comment { get; set; }
        public string CancelFlag { get; set; }
        public DateTime? CreateDate { get; set; }
        public decimal? CreateBy { get; set; }
        public DateTime? UpdateDate { get; set; }
        public decimal? UpdateBy { get; set; }
        public decimal? Status { get; set; }
        public string Guid { get; set; }
        public string FarmGuid { get; set; }
    }
}

File: App2/src/app/_core/_model/nutrition.ts
export interface Nutrition {
  id: number;
  nutritionType: string;
  nutritionNo: string;
  nutritionName: string;
  nutritionElement: string;
  nutritionEffect: string;
  nutritionSideEffect: string;
  nutritionBreed: string;
  nutritionRange: string;
  nutritionCare: string;
  comment: string;
  cancelFlag: string;
  createDate: string | null;
  createBy: number | null;
  updateDate: string | null;
  updateBy: number | null;
  status: number | null;
  guid: string;
  farmGuid: string;
  nutritionTypeName: string;

}

